fec/fec/static/js/calendar.js

```diff
@@ -234,14 +242,31 @@ Calendar.prototype.styleButtons = function() {
 
 Calendar.prototype.handleRender = function(view) {
   $(document.body).trigger($.Event('calendar:rendered'));
+  this.highlightToday();
 };
 
+Calendar.prototype.handleDayRender = function(date, cell) {
+  if (date.date() === 1) {
+    var firstIndex = cell.index()
+    cell.append(date.format('MMMM'));
+  }
+}
+
 Calendar.prototype.handleEventClick = function(calEvent, jsEvent, view) {
-  var $eventContainer = $(jsEvent.target).closest('.fc-event-container');
+  var $eventContainer = $(jsEvent.target).closest('.fc-content');
   var tooltip = new CalendarTooltip(templates.details(calEvent));
   $eventContainer.append(tooltip.$content);
 };
 
+Calendar.prototype.highlightToday = function() {
+  var $today = $('thead .fc-today')
```

Comment:
It only kind of does. It highlights the `td` in the `thead` for that week, but the rest of the `td`s below don't have any class or anything applied, so it would just highlight the top part of the cell. It's clunky, so if you think there's a better way to do it, I'd love to.